integration_tests/sdk/no_return_test.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+from utils import run_flow_test
+
+from aqueduct import op
+
+
+@op
+def no_return() -> None:
+    return None
+
+
+@pytest.mark.publish
+def test_operator_with_no_return(client):
+    result = no_return()
+    assert result.get() is None
+    flow = run_flow_test(client, artifacts=[result], delete_flow_after=False)
```

Comment:
nit: can we put this in the try-statement? That way if it publishes but fails, we still delete it. We have a lot of tests like this actually, but I'm planning a refactor of the test utils soon.